python/ray/train/BUILD

```diff
@@ -71,23 +71,154 @@ py_test(
 py_test(
     name = "tune_cifar_torch_pbt_example",
     size = "medium",
-    main = "examples/tune_cifar_torch_pbt_example.py",
-    srcs = ["examples/tune_cifar_torch_pbt_example.py"],
+    main = "examples/pytorch/tune_cifar_torch_pbt_example.py",
+    srcs = ["examples/pytorch/tune_cifar_torch_pbt_example.py"],
     tags = ["team:ml", "exclusive", "pytorch", "tune"],
     deps = [":train_lib"],
     args = ["--smoke-test"]
 )
 
 py_test(
-    name = "tune_linear_example",
+    name = "tune_torch_regression_example",
     size = "small",
-    main = "examples/tune_linear_example.py",
-    srcs = ["examples/tune_linear_example.py"],
+    main = "examples/pytorch/tune_torch_regression_example.py",
+    srcs = ["examples/pytorch/tune_torch_regression_example.py"],
     tags = ["team:ml", "exclusive", "tune"],
     deps = [":train_lib"],
     args = ["--smoke-test"]
 )
 
+# Formerly AIR examples
+
+py_test(
+    name = "distributed_sage_example",
+    size = "small",
+    main = "examples/pytorch_geometric/distributed_sage_example.py",
+    srcs = ["examples/pytorch_geometric/distributed_sage_example.py"],
+    tags = ["team:ml", "exclusive", "gpu"],
+    deps = [":train_lib"],
+    args = ["--use-gpu", "--num-workers=2", "--epochs=1", "--dataset=fake"]
+)
+
+py_test(
+    name = "horovod_cifar_pbt_example",
```

Comment:
do we need an `__init__.py` in the ray/train/examples/horovod?

Reply:
We may run into issues with imports (and we are importing functions from the examples) without it.